CONTRIBUTING.md

```diff
@@ -96,11 +95,6 @@ Building and testing are documented in our
 
     _Note that you are allowed to force push into your development branches._
 
-- **Use draft pull requests for work-in-progress:**
-
-  - The draft state signals that the code is not ready for review, but still
-    gives a nice URL to track the ongoing work.
-
```

Comment:
This is an obvious feature of github, being used as intended. Trying to keep this document not overwhelming.